mmcv/tensorrt/tensorrt_utils.py

```diff
@@ -65,10 +84,11 @@ def parse_data(name, typ):
 
             if len(node_inputs) >= 3:
                 max_output_boxes_per_class = parse_data(
-                    node_inputs[2], np.int64)
+                    node_inputs[2], np.int64, max_output_boxes_per_class)
```

Comment:
Here `max_output_boxes_per_class` should become static. But if `max_num` is set by `bbox.shape`, then it may be too large for TensorRT to allocate memory for NMS plugin.

Reply:
If `max_output_boxes_per_class` is not constant, then the default value (1000000) will be inserted instead, and unnecessary nodes will be removed from the ONNX graph before exporting to TensorRT. The default value of the constant can be decreased.